app/models/encounter.rb

```diff
@@ -1,12 +1,13 @@
 class Encounter < ApplicationRecord
   include Mergeable
+  include SQLHelpers
 
   belongs_to :patient, optional: true
   belongs_to :facility
 
   has_many :observations
-  has_many :blood_pressures, through: :observations, source: :observable, source_type: 'BloodPressure'
   has_many :blood_sugars, through: :observations, source: :observable, source_type: 'BloodSugar'
+  has_many :blood_pressures, through: :observations, source: :observable, source_type: 'BloodPressure'
```

Comment:
Why this change?

Reply:
Reverted. I was using BloodSugars before BloodPressures everywhere so I changed it about. It's not a useful change.